conftest.py

```diff
@@ -237,3 +239,54 @@ def remote_file(request):
 
     session.delete(new_connection)
     session.flush()
+
+
+@pytest.fixture
+def database_table_fixture(request):
```

Comment:
Do we need to have the word `fixture` in the name?

Reply:
Indeed, @dimberman , we don't need the suffix `fixture`!

So far, we have not been following a convention to name our fixtures. One of them is called `test_table`, which is confusing - since it is prefixed just like the tests. And we have other fixtures which don't have a prefix or suffix (e.g. `output_table` and `sql_server.`)

Having some naming convention would help users unfamiliar with the code base quickly identify if a particular test argument is a fixture or a normal parameter.

That said, I'm fine dropping that if the rest of the team prefers it :)